pywbem/_subscription_manager.py

```diff
@@ -1200,6 +1244,9 @@ def _create_destination(self, server_id, dest_url, owned):
         dest_inst['CreationClassName'] = DESTINATION_CLASSNAME
         dest_inst['SystemCreationClassName'] = SYSTEM_CREATION_CLASSNAME
         dest_inst['SystemName'] = self._systemnames[server_id]
+        # Populate this property only if value provided
+        if persistence_type:
```

Comment:
This condition does not set it if the value is 0, which is in conflict with the parameter description.

Reply:
Changed to test for None

DONE

Reply:
A resolution to this issue is not in the latest version of the code as of 7/4 (commit b7175d3).

Reply:
In the last update to this PR, the change is in.